README.md

```diff
@@ -30,14 +30,18 @@ targets: '[{"project": "Backlog", "section": "Development Done"}, {"project": "C
 ```
 if you don't want to move task omit `targets`.
 
-
+## Sample PR Description
+``
+**Asana Task:** [Task Name](https://app.asana.com/0/1/2)
+``
+*Always leave a blank space between the trigger phrase & the asana url.*
```

Comment:
why is this needed?

Reply:
Due to that particular issue, mentioned above.

Reply:
Removed after making the space optional.